beta-src/src/state/game/game-api-slice.ts

```diff
@@ -293,13 +297,41 @@ const drawBuilds = (state) => {
   }
 };
 
+const drawDislodged = (state) => {
+  const {
+    data: {
+      data: { contextVars, currentOrders, units },
+    },
+    ordersMeta,
+    territoriesMeta,
+    overview: { members, phase },
+  }: {
+    data;
+    ordersMeta: OrdersMeta;
+    territoriesMeta: TerritoriesMeta;
+    overview: {
+      members: GameOverviewResponse["members"];
+      phase: GameOverviewResponse["phase"];
+    };
+  } = current(state);
+  if (phase === "Retreats") {
+    currentOrders.forEach((order, index) => {
+      const command: GameCommand = {
+        command: "DISLODGED",
+      };
+      setCommand(state, command, "unitCommands", order.unitID);
+    });
+  }
+};
+
```

Comment:
In game API slice you are calling this function anytime the `ordersMeta` property is updated. This means if you update an order to a disband icon this code will change the icon back to dislodged. 
![Screen Shot 2022-04-26 at 3 23 12 PM](https://user-images.githubusercontent.com/93741108/165394846-8137b329-5805-40b6-87cb-1116aa6f7632.png)

Reply:
@bryancodazen correct me if I am wrong but I believe @jasonekstrom you need to check to make sure there are no ready orders in game status, or saved orders in ordersMeta (checking its saved property) pertaining to the unit in order before setting its ui state to dislodge. This is because whenever you process either a move or hold order through clicking a terr in retreat, it would remove dislodge state, and the user would then save it which would break the condition mentioned above. If they refresh before save, icon goes back to dislodge state because order was never saved.

Reply:
Updated in my newest commit